ui/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "appx-ui",
   "version": "0.1.0",
-  "proxy_server": "http://localhost:3000/",
```

Comment:
This change does not seem necessary, please kindly test functionality by removing this change

Reply:
Tested. Change unnecessary. Reverted